buildsystem/codecompliance/authors.py

```diff
@@ -6,6 +7,45 @@
 
 import re
 
+def get_email_obfuscation_substitutions():
+    """
+    What tokens in email should be replaced with
+    """
+    return {
+    '.' : ' noot ',
+    '@' : ' à ',
+    'gmail' : '!gm!',
+    'com' : '!corn!',
```

Comment:
okay, I was trying to hide "gmail", since bots may search for this if they are brave enough to attempt parsing an obfuscated address. maybe that's worrying too much